components/eam/src/physics/cam/aer_rad_props.F90

```diff
@@ -418,7 +420,7 @@ subroutine aer_rad_props_lw(is_cmip6_volc, list_idx, state, pbuf,  odap_aer)
 
    ! Contributions from modal aerosols.
    if (nmodes > 0) then
-      call modal_aero_lw(list_idx, state, pbuf, odap_aer)
+      call modal_aero_lw(list_idx, dt, state, pbuf, odap_aer)
```

Comment:
The addition of dummy variable "dt" does break the compilation of rrtmgp/cam_optics.F90 in its call to aer_rad_props_sw at line 506

```
call aer_rad_props_sw(icall, state, pbuf, &
                            count(night_indices > 0), night_indices, is_cmip6_volc, &
                            tau, tau_w, tau_w_g, tau_w_f)
```
Similar changes as in rrtmg/radiation.F90 is needed.

Reply:
Thanks @wlin7 for your quick review. I haven't touched rrtmgp codes yet as I will have to go through the rrtmgp submodule for making those changes. I  am actively working on this PR. It touches a lot of interdependent codes so I will do rigorous testing before it is ready for merging to master.

In you last line, you mean "rrtmg**p**", right? "rrtmg" codes looks okay to me.

Reply:
@singhbalwinder , in that last line, I did mean rrtmg as a reference. Yes, the rrtmg version is ok. I meant to make a change to rrtmgp/cam_optics.F90 similar to what you did in rrtmg/radiation.F90. Sorry for the confusion.